apps/web/utils/outlook/thread.ts

```
@@ -0,0 +1,174 @@
+import type { ParsedMessage } from "@/utils/types";
+import { createScopedLogger } from "@/utils/logger";
+
+const logger = createScopedLogger("outlook/thread");
+
+export async function getThread(
+  threadId: string,
+  client: OutlookClient,
+): Promise<Message[]> {
+  const messages = await client
+    .getClient()
+    .api(`/me/messages`)
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)